### PR TITLE
No submodule plugins

### PR DESCRIPTION
**Pull request by Schneegans**
There we go. Once merged I would archive all plugin repositories to make them read-only. Once this change is in `master` we may delete them...

### CODE REVIEW

**Comment by TheJhonny007** [comment]
Should we delete them? Or should we keep them for the git history?

**Comment by Schneegans** [comment]
I think we should keep them around for some time. Once we are confident that we will never need them, I think we could delete them.

File: plugins/csp-anchor-labels/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Anchor Labels for CosmoScout VR
+
+A CosmoScout VR plugin which draws a clickable label at each celestial anchor. When activated, the user automatically travels to the according body. The size and overlapping-behavior of the labels can be adjusted. This plugin is built as part of CosmoScout's build process. See the [main repository](https://github.com/cosmoscout/cosmoscout-vr) for instructions.
+
+This is a default plugin of CosmoScout VR. Hence, any **issues should be reported to the [main issue tracker](https://github.com/cosmoscout/cosmoscout-vr/issues)**. There you can add a label indicating which plugins are affected.
```
Comment (TheJhonny007) [inline plugins/csp-anchor-labels/README.md:5]:
```suggestion
A CosmoScout VR plugin which draws a clickable label at each celestial anchor. When activated, the user automatically travels to the according body. The size and overlapping-behavior of the labels can be adjusted.
```

File: plugins/csp-atmospheres/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Atmospheres for CosmoScout VR
+
+A CosmoScout VR plugin for drawing atmospheres around celestial bodies. It calculates single Mie- and Rayleigh scattering via raycasting in real-time. This plugin is built as part of CosmoScout's build process. See the [main repository](https://github.com/cosmoscout/cosmoscout-vr) for instructions.
+
+This is a default plugin of CosmoScout VR. Hence, any **issues should be reported to the [main issue tracker](https://github.com/cosmoscout/cosmoscout-vr/issues)**. There you can add a label indicating which plugins are affected.
```
Comment (TheJhonny007) [inline plugins/csp-atmospheres/README.md:5]:
```suggestion
A CosmoScout VR plugin for drawing atmospheres around celestial bodies. It calculates single Mie- and Rayleigh scattering via raycasting in real-time.
```

File: plugins/csp-custom-web-ui/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
+# Custom Web UI for CosmoScout VR
+
+A CosmoScout VR plugin which allows adding custom HTML-based UI elements as sidebar-tabs, as floating windows or into free space. This plugin is built as part of CosmoScout's build process. See the [main repository](https://github.com/cosmoscout/cosmoscout-vr) for instructions.
+
+This is a default plugin of CosmoScout VR. Hence, any **issues should be reported to the [main issue tracker](https://github.com/cosmoscout/cosmoscout-vr/issues)**. There you can add a label indicating which plugins are affected.
```
Comment (TheJhonny007) [inline plugins/csp-custom-web-ui/README.md:5]:
```suggestion
A CosmoScout VR plugin which allows adding custom HTML-based UI elements as sidebar-tabs, as floating windows or into free space.
```

File: plugins/csp-fly-to-locations/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+# Fly-To-Locations for CosmoScout VR
+
+A CosmoScout VR plugin which adds several quick travel targets to the sidebar. It supports shortcuts to celestial bodies and to specific geographic locations on those bodies. This plugin is built as part of CosmoScout's build process. See the [main repository](https://github.com/cosmoscout/cosmoscout-vr) for instructions.
+
+This is a default plugin of CosmoScout VR. Hence, any **issues should be reported to the [main issue tracker](https://github.com/cosmoscout/cosmoscout-vr/issues)**. There you can add a label indicating which plugins are affected.
```
Comment (TheJhonny007) [inline plugins/csp-fly-to-locations/README.md:5]:
```suggestion
A CosmoScout VR plugin which adds several quick travel targets to the sidebar. It supports shortcuts to celestial bodies and to specific geographic locations on those bodies.
```

File: plugins/csp-lod-bodies/README.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
+# Level-of-detail bodies for CosmoScout VR
+
+A CosmoScout VR plugin wich draws level-of-detail planets and moons. This plugin supports the visualization of entire planets in a 1:1 scale. The data is streamed via Web-Map-Services (WMS) over the internet. A dedicated MapServer is required to use this plugin.
+
+This plugin is built as part of CosmoScout's build process. See the [main repository](https://github.com/cosmoscout/cosmoscout-vr) for instructions.
+
+This is a default plugin of CosmoScout VR. Hence, any **issues should be reported to the [main issue tracker](https://github.com/cosmoscout/cosmoscout-vr/issues)**. There you can add a label indicating which plugins are affected.
```
Comment (TheJhonny007) [inline plugins/csp-lod-bodies/README.md:7]:
```suggestion
```

File: plugins/csp-measurement-tools/README.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
+# Measurement tools for CosmoScout VR
+
+A CosmoScout VR plugin with several tools for terrain measurements. For example, it supports the measurement of distances, height profiles, volumes or areas. The plugin is built as part of CosmoScout's build process. See the [main repository](https://github.com/cosmoscout/cosmoscout-vr) for instructions.
+
+This is a default plugin of CosmoScout VR. Hence, any **issues should be reported to the [main issue tracker](https://github.com/cosmoscout/cosmoscout-vr/issues)**. There you can add a label indicating which plugins are affected.
```
Comment (TheJhonny007) [inline plugins/csp-measurement-tools/README.md:5]:
```suggestion
A CosmoScout VR plugin with several tools for terrain measurements. For example, it supports the measurement of distances, height profiles, volumes or areas.
```

File: plugins/csp-minimap/README.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
+# Minimap for CosmoScout VR
+
+A CosmoScout VR plugin which shows a 2D-Minimap in the user interface. This plugin is built as part of CosmoScout's build process. See the [main repository](https://github.com/cosmoscout/cosmoscout-vr) for instructions.
+
+This is a default plugin of CosmoScout VR. Hence, any **issues should be reported to the [main issue tracker](https://github.com/cosmoscout/cosmoscout-vr/issues)**. There you can add a label indicating which plugins are affected.
```
Comment (TheJhonny007) [inline plugins/csp-minimap/README.md:5]:
```suggestion
A CosmoScout VR plugin which shows a 2D-Minimap in the user interface.
```

File: plugins/csp-rings/README.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
+# Rings for CosmoScout VR
+
+A CosmoScout VR plugin which can draw simple rings around celestial bodies. The rings can be configured with a inner and a outer radius and a texture. This plugin is built as part of CosmoScout's build process. See the [main repository](https://github.com/cosmoscout/cosmoscout-vr) for instructions.
+
+This is a default plugin of CosmoScout VR. Hence, any **issues should be reported to the [main issue tracker](https://github.com/cosmoscout/cosmoscout-vr/issues)**. There you can add a label indicating which plugins are affected.
```
Comment (TheJhonny007) [inline plugins/csp-rings/README.md:5]:
```suggestion
A CosmoScout VR plugin which can draw simple rings around celestial bodies. The rings can be configured with a inner and a outer radius and a texture.
```

File: plugins/csp-satellites/README.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
+# Satellites for CosmoScout VR
+
+A CosmoScout VR plugin which draws GTLF models at positions based on SPICE data. It uses physically based rendering for surface shading. This plugin is built as part of CosmoScout's build process. See the [main repository](https://github.com/cosmoscout/cosmoscout-vr) for instructions.
+
+This is a default plugin of CosmoScout VR. Hence, any **issues should be reported to the [main issue tracker](https://github.com/cosmoscout/cosmoscout-vr/issues)**. There you can add a label indicating which plugins are affected.
```
Comment (TheJhonny007) [inline plugins/csp-satellites/README.md:5]:
```suggestion
A CosmoScout VR plugin which draws GTLF models at positions based on SPICE data. It uses physically based rendering for surface shading.
```

File: plugins/csp-sharad/README.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
+# Sharad rendering for CosmoScout VR
+
+A CosmoScout VR plugin which renders radar datasets acquired by the Mars Reconnaissance Orbiter. The SHARAD profiles are rendered inside of Mars, the Martian surface is made translucent in front of the profiles. This plugin is built as part of CosmoScout's build process. See the [main repository](https://github.com/cosmoscout/cosmoscout-vr) for instructions.
+
+This is a default plugin of CosmoScout VR. Hence, any **issues should be reported to the [main issue tracker](https://github.com/cosmoscout/cosmoscout-vr/issues)**. There you can add a label indicating which plugins are affected.
```
Comment (TheJhonny007) [inline plugins/csp-sharad/README.md:5]:
```suggestion
A CosmoScout VR plugin which renders radar datasets acquired by the Mars Reconnaissance Orbiter. The SHARAD profiles are rendered inside of Mars, the Martian surface is made translucent in front of the profiles.
```

File: plugins/csp-simple-bodies/README.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
+# Simple bodies for CosmoScout VR
+
+A CosmoSout VR plugin which renders simple spherical celestial bodies. The bodies are drawn as an ellipsoid with an equirectangular texture. This plugin is built as part of CosmoScout's build process. See the [main repository](https://github.com/cosmoscout/cosmoscout-vr) for instructions.
+
+This is a default plugin of CosmoScout VR. Hence, any **issues should be reported to the [main issue tracker](https://github.com/cosmoscout/cosmoscout-vr/issues)**. There you can add a label indicating which plugins are affected.
```
Comment (TheJhonny007) [inline plugins/csp-simple-bodies/README.md:5]:
```suggestion
A CosmoSout VR plugin which renders simple spherical celestial bodies. The bodies are drawn as an ellipsoid with an equirectangular texture.
```

File: plugins/csp-stars/README.md [hunk 12]
```diff
@@ -0,0 +1,57 @@
+# Stars for CosmoScout VR
+
+A CosmoSout VR plugin which draws 3D-stars loaded from catalogues. For now, it supports the Tycho, the Tycho2 and the Hipparcos catalogue. This plugin is built as part of CosmoScout's build process. See the [main repository](https://github.com/cosmoscout/cosmoscout-vr) for instructions.
+
+This is a default plugin of CosmoScout VR. Hence, any **issues should be reported to the [main issue tracker](https://github.com/cosmoscout/cosmoscout-vr/issues)**. There you can add a label indicating which plugins are affected.
```
Comment (TheJhonny007) [inline plugins/csp-stars/README.md:5]:
```suggestion
A CosmoSout VR plugin which draws 3D-stars loaded from catalogues. For now, it supports the Tycho, the Tycho2 and the Hipparcos catalogue.
```

File: plugins/csp-trajectories/README.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
+# Trajectories for CosmoScout VR
+
+A CosmoSout VR plugin which draws trajectories of celestial bodies and spacecrafts. The color, length, number of samples and reference frame can be configured. This plugin is built as part of CosmoScout's build process. See the [main repository](https://github.com/cosmoscout/cosmoscout-vr) for instructions.
+
+This is a default plugin of CosmoScout VR. Hence, any **issues should be reported to the [main issue tracker](https://github.com/cosmoscout/cosmoscout-vr/issues)**. There you can add a label indicating which plugins are affected.
```
Comment (TheJhonny007) [inline plugins/csp-trajectories/README.md:5]:
```suggestion
A CosmoSout VR plugin which draws trajectories of celestial bodies and spacecrafts. The color, length, number of samples and reference frame can be configured.
```

File: plugins/csp-web-api/README.md [hunk 14]
```diff
@@ -0,0 +1,27 @@
+# Web API for CosmoScout VR
+
+A CosmoScout VR plugin which provides an HTTP protocol for controlling the application. This plugin is built as part of CosmoScout's build process. See the [main repository](https://github.com/cosmoscout/cosmoscout-vr) for instructions.
+
+This is a default plugin of CosmoScout VR. Hence, any **issues should be reported to the [main issue tracker](https://github.com/cosmoscout/cosmoscout-vr/issues)**. There you can add a label indicating which plugins are affected.
```
Comment (TheJhonny007) [inline plugins/csp-web-api/README.md:5]:
```suggestion
A CosmoScout VR plugin which provides an HTTP protocol for controlling the application.
```

**Pull request merged**